pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MultiStageOperator.java

```diff
@@ -64,8 +64,9 @@ public TransferableBlock nextBlock() {
     try (InvocationScope ignored = Tracing.getTracer().createScope(getClass())) {
       _operatorStats.startTimer();
       TransferableBlock nextBlock = getNextBlock();
+      _operatorStats.endTimer(nextBlock);
```

Comment:
we dont have any tests for MultiStageOperator stats and trace. it is good to add a collection of these